SPL/SPL_c1.php

```diff
@@ -285,10 +302,12 @@ public function setFileClass(#[LanguageLevelTypeAware(['8.0' => 'string'], defau
 
     /**
      * Sets the class used with getFileInfo and getPathInfo
+     * @template T of SplFileInfo*
```

Comment:
```suggestion
     * @template T of SplFileInfo
```
no ?

I never saw a syntax with `*`